WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItemDescriptionInfoDialog.swift

```diff
@@ -0,0 +1,86 @@
+                                        Image(systemName: "arrow.up.forward.square")
+                                            .resizable()
+                                            .frame(width: Layout.externalLinkImageSize * scale, height: Layout.externalLinkImageSize * scale)
+                                    }
+                                }
+                                .buttonStyle(PrimaryButtonStyle())
+                                .safariSheet(isPresented: $showLearnMoreWebView, url: WooConstants.URLs.shippingCustomsInstructionsForEUCountries.asURL())
+
+                                Button {
+                                    dismiss()
+                                } label: {
+                                    Text(Localization.doneButtonTitle)
+                                }
+                                .buttonStyle(SecondaryButtonStyle())
+                            }
+                            .padding(Layout.outterPadding)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .background(Color(.systemBackground))
+                            .cornerRadius(Layout.cornerRadius)
+                            .frame(width: geometry.size.width)
+                            .frame(minHeight: geometry.size.height)
+                        }
+                }
+            }
+            .padding(Layout.outterPadding)
+            .frame(maxWidth: .infinity, alignment: .center)
+        }
+    }
+}
+extension WooShippingCustomsItemDescriptionInfoDialog {
+    enum Localization {
+        static let title = NSLocalizedString("Description", comment: "Title for the custom description educational dialog")
+        static let bodyParagraph = NSLocalizedString("When shipping to countries that follow European Union (EU) customs rules, " +
+                                                     "you must provide a clear, specific description on every item. " +
+                                                     "For example, if you are sending clothing, you must indicate what type of clothing" +
+                                                     " (e.g. men's shirts, girl's vest, boy's jacket) for the description to be acceptable." +
+                                                     " Otherwise, shipments may be delayed or interrupted at customs.",
+                                                          comment: "Body for the custom items description educational dialog")
+        static let learnMoreButtonTitle = NSLocalizedString("Learn more",
+                                                                      comment: "Button title for the learn more action in the custom descriptions info dialog")
+        static let doneButtonTitle = NSLocalizedString("Done",
+                                                       comment: "Button title for the done button in the customs description educational dialog")
```

Comment:
Could you update these strings to use the [reverse-DNS naming style for keys](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys), instead of the English strings as keys?

Reply:
Good catch! Done in 58f965c70ffbfe4a2bc37ef0983d62045d256977